public/docs/claude-guide.html

```
@@ -0,0 +1,1572 @@
+  .hdr,.nav,.nav-overlay,#scrolltop,.menu-btn { display: none !important; }
+  .layout { padding-top: 0; }
+  .main { margin-left: 0; padding: 16px; }
+  body { background: #fff; color: #000; }
+}
+</style>
+</head>
+<body>
+
+<!-- ── Header ─────────────────────────────────────────── -->
+<header class="hdr">
+  <button class="menu-btn" id="menuBtn" aria-label="Toggle navigation">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)